### ISSUE TITLE
CLEARTEXT communication not permitted by network security policy

### ISSUE THREAD

**Issue by cookieguru** [eclipsesource/tabris-js#1857]
### Problem description

When running `tabris serve --logging` and then attempting to connect with an Android 9 device, the following error is seen on the device:

```plain
CLEARTEXT communication to 192.168.1.x not permitted by network security policy
```

The server's request log shows no requests.

Opening `http://192.168.1.x:8080/` in a browser on the same Android device shows:
```
Tabris.js CLI version 3.0.0-rc1-dev.20190416 is running
```

### Environment

- Tabris.js version: `3.0.0-rc1-dev.20190416`
- Device: Pixel 3
- OS: Android 9

---

The APK was generated from a bare project created with `tabris init` and then built using the cloud build service without adjusting any of the build settings

**Comment by cookieguru** [comment]
I can successfully connect to the aformentioned server with the `3.0.0-beta2` Developer app currently published in the Play Store

**Comment by mpost** [comment]
You are seeing the expected behavior. Android prohibits clear text communication by default. We did not opt to remove this security feature in tabris. You can use a custom security policy on your app or a "global" flag in your `config.xml` like: 

```  <platform name="android">
    <edit-config file="app/src/main/AndroidManifest.xml" target="/manifest/application" mode="merge">
      <application android:usesCleartextTraffic="true" />
    </edit-config>
```

**Comment by cookieguru** [comment]
Hm, this prevents a bit of a dilemma.  The solution for this isn't documented anywhere, which prevents `tabris serve` from working.
I don't think it would be advisable to silently enable the flag on debug builds, as it would give the impression of non-https calls working when in fact they would fail on production builds.
Adding a cert to the tabris-cli would be difficult due to the number of addresses that could be used (and there usually isn't a domain name available for an IP address).

**Comment by mpost** [comment]
I agree with the current confusion. tabris ios behaves in the same way. It is does not allow cleartext communication by default. Also the app created by `tabris init` does not allows cleartext communication as can be seen [here](https://github.com/eclipsesource/generator-tabris-js/blob/master/app/templates/cordova/config.xml).

The ios client however respects the `config.xml` element [`access`](https://cordova.apache.org/docs/en/latest/config_ref/#access). It would make sense to map this element to the android based `<network-security-config>`.
